backtesting/backtesting.py

```diff
@@ -707,12 +705,12 @@ def new_order(self,
 
         if is_long:
             if not (sl or -np.inf) <= (limit or stop or self.last_price) <= (tp or np.inf):
-                raise ValueError("Long orders require: SL ({}) < LIMIT ({}) < TP ({})".format(
-                    sl, limit or stop or self.last_price, tp))
+                raise ValueError(f"Long orders require: "
+                      f"SL ({sl}) < LIMIT ({limit or stop or self.last_price}) < TP ({tp})")
```

Comment:
```suggestion
                raise ValueError(
                    "Long orders require: "
                    f"SL ({sl}) < LIMIT ({limit or stop or self.last_price}) < TP ({tp})")
```
The usual style guidelines say either hanging or indented, but not mixed. Thought flake8 catches that. :thinking: